policies/accepted/9.md

```diff
@@ -0,0 +1,9 @@
+# Outside Collaborators
+
+There is a need to define a label for and track outside collaborators that may require specific access to org resources. This can include users who need the write permission to org-owned repos while not being a member of the Cog-Creators org. A possible example for this definition includes large Red-related projects that may not yet be part of the core bot, where the org would like to foster development of said project and/or work closely with the collaborator(s).
```

Comment:
```suggestion
There is a need to define a label for and track outside collaborators that may require specific access to org controlled github repositories. This can include users who need the write permission to org-owned repos while not being a member of the Cog-Creators org. A possible example for this definition includes large Red-related projects that may not yet be part of the core bot, where the org would like to foster development of said project and/or work closely with the collaborator(s).
```